### DISCUSSION TITLE
Incoming release

### DISCUSSION THREAD

**Discussion by lothesven-clauger** [BeanieODM/beanie#1031] [General]
Would it be possible for us users around here to have a approximation of time for next release ? Like "expected in the next week/month"

I am really glad a feature request have gone through the process and is merged to the project and am waiting for the release. Thank you for the work :)

I have no clue wether that release would be in days, week or months and a lot of my development work can rely on that. I would handle it differently if I knew a release is near or not.

**Comment by LawMixer** [comment]
No release ETA, but we're making our rounds through all of the GitHub pull requests and issues.

Thanks for staying patient!
